views/orgm/opportunity.view.lkml

```diff
@@ -101,6 +108,34 @@ view: opportunity {
   # Dimensions
   #
 
+
+  dimension: cpq_renewal_arr__c {
```

Comment:
side note (for a future refactoring) - wondering why for dimensions we have `type`, `sql`, `description` ordering of fields and for measures we have a different `description`,`label`,`sql`,`type` ordering.